packages/flutter_test/lib/src/controller.dart

```diff
@@ -23,6 +23,202 @@ const double kDragSlopDefault = 20.0;
 
 const String _defaultPlatform = kIsWeb ? 'web' : 'android';
 
+/// Class that programatically interacts with the [Semantics] tree.
+///
+/// Allows for testing of the [Semantics] tree, which is used by accessibility
```

Comment:
nit: I noticed you changed "accessibility tools" to "assistive technologies" above in basic.dart. Maybe we should be consistent here as well?

Reply:
Good catch! I think I missed this one when searching for "accessibility tools" because of the line break. Updated!